taxi/urls.py

```diff
@@ -53,13 +53,25 @@
     ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "drivers/<int:pk>/",
+        DriverDetailView.as_view(),
+        name="driver-detail"
+    ),
+    path(
+        "drivers/",
+        DriverListView.as_view(),
+        name="driver-list"
+    ),
```

Comment:
This URL pattern for `DriverListView` is a duplicate of the one defined earlier. Duplicate URL patterns can cause conflicts and unexpected behavior. Please remove or modify this duplicate entry.